test/WebSites/ModelBindingWebSite/Controllers/VehicleController.cs

```diff
@@ -37,7 +37,7 @@ public class VehicleController : Controller
         [HttpPut("/api/vehicles/{id}")]
         [Produces("application/json")]
         public object UpdateVehicleApi(
-            [Range(1, 500)] int id,
```

Comment:
This wasn't really ever intended to work, but we did it.... There's a few of these that need to be cleaned up.